deploy.ps1

```diff
@@ -18,6 +18,7 @@ $TemplateDir            = "${BaseDirWithoutQuotes}template"
 $UIMovieDir             = "${BaseDirWithoutQuotes}UIMovies"
 
 # create output directory structure
+Remove-Item $DeployDir -Recurse
```

Comment:
removes the temp deploy directory for any lingering files. I noticed that this would sometimes be filled with old dlls